modules/sorts/test/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+add_executable(merge_sort_test merge_sort_test.cpp)
+target_link_libraries(merge_sort_test Sorts)
+
+# Tests
+add_test(
+    NAME "Merge_Sort_Test"
+    COMMAND merge_sort_test --log-format=XML --log-sink=TEST-merge_sort_test.xml --log-level=all --report-level=no
```

Comment:
One final thing that's my fault, this should be:
`--log_format=XML --log_sink=TEST-merge_sort_test.xml --log_level=all --report_level=no`
Underscores rather than hyphens in the arguments; I put it incorrectly on the guide on docs.

Though I think it would be useful for boost to cause the test to fail if passed invalid arguments considering that at the moment it's ignoring the invalid arguments and not producing any sort of report.